module/VuFind/src/VuFind/Form/Form.php

```diff
@@ -366,6 +370,18 @@ protected function getFormSettingFields()
         ];
     }
 
+    /**
+     * Return a list of field names to read from form element settings.
+     *
+     * @return array
+     */
+    protected function getFormElementSettingFields()
+    {
+        return [
+            'required', 'help','value', 'inputType', 'group', 'placeholder'
```

Comment:
There's a missing space after 'help' -- not a big deal, but might be worth cleaning up for style's sake. :-)

Reply:
Yes, corrected.